packages/theme/css/characteristics/actionable-next.css

```diff
@@ -9,9 +9,11 @@
   --salt-actionable-cta-background-active: var(--salt-palette-accent-action-active);
   --salt-actionable-cta-background-disabled: var(--salt-palette-accent-disabled);
   --salt-actionable-cta-borderColor: var(--salt-palette-accent);
-  --salt-actionable-cta-borderColor-hover: transparent; /* TBD when adding bordered button */
+  --salt-actionable-cta-borderColor-hover: transparent;
+  /* TBD when adding bordered button */
```

Comment:
I'm assuming all of these TBD comments can be removed?

Reply:
removed